src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -43,6 +43,9 @@
  * Summaries that contain a non-empty {@code {@return}} are allowed.
  * Check also violate Javadoc that does not contain first sentence, though with {@code {@return}} a
  * period is not required as the Javadoc tool adds it.
+ * Note: In past, summary could only be implemented using the first sentence of each Javadoc.
+ * Now,we can also denote summary by the corresponding @summary tag. We support both
```

Comment:
```suggestion
 * Now, we can also denote summary by the corresponding @summary tag. We support both
```
little typo space missing

Reply:
I would avoid `past ` and `now` words.
We just need to define what is summary. We have 2 options now.

Reply:
Done